src/backend/columnar/sql/columnar--10.2-4--11.1-1.sql

```diff
@@ -1,3 +1,3 @@
 -- columnar--10.2.4--11.1-1.sql
 
--- placeholder, no changes 
\ No newline at end of file
+#include "udfs/alter_table_set_access_method/11.1-1.sql"
```

Comment:
`11.1-1` is already installed, so we can't use that. We should make this run when upgrading, either to `11.1-2` or `11.2-1`.